src/main/java/com/igot/cb/designation/controller/DesignationController.java

```diff
@@ -30,6 +36,19 @@ public ResponseEntity<String> loadJobsFromExcel(@RequestParam("file") MultipartF
     }
   }
 
+  @GetMapping("/read/{id}")
+  public ResponseEntity<?> playListRead(@PathVariable String id) {
+    CustomResponse response = designationService.readDesignation(id);
+    return new ResponseEntity<>(response, response.getResponseCode());
+  }
+
   //update API to store the refNodes
 
+  @PutMapping("/update/designation")
```

Comment:
use the proper rest path... 

to the controller class -- it should be -> "/designation/v1"
to the methods -> "/update"

Reply:
made the changes